c/tskit/core.h

```diff
@@ -510,18 +536,19 @@ tsk_size_t tsk_search_sorted(const double *array, tsk_size_t size, double value)
 double tsk_round(double x, unsigned int ndigits);
 
 /**
-@brief Check if a number is TSK_UNKNOWN_TIME
+@brief Check if a number is :c:macro:`TSK_UNKNOWN_TIME`
```

Comment:
The ``@brief`` doesn't support rst cross references (nor does return, param etc). Any cross refs like this must be in an ``@rst`` block.

Reply:
How annoying! I've tried a few different ways around this, but have resorted to unlinking the macros in the params :(

Reply:
Yep. Let me know when it's ready for a final look.